### PR TITLE
Ndbex 63626 pensions review page

### PR DESCRIPTION
**Pull request by Midge-dev**
## Summary

_(Summarize the changes that have been made to the platform)_
- Replace  the online Pensions form with approved review page letting Veterans know the form isn't currently working and they can complete by mail.

_(What is the solution, why is this the solution)_
- The purpose of this page is to dynamically display any "In progress" form data to the Veteran so they can refer to it when filling out the paper form. This page is temporary while this online form is updated.

_(Which team do you work for, does your team own the maintenance of this component?)_
- Non-disability Benefits Experience

## Related issue(s)

[Pension | Build Review Page #63626](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63626)

## Testing done

- Manual testing for static elements
- Accessibility

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |   image  |   image |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by Midge-dev** [comment]
This is ready for review. Thanks!

@department-of-veterans-affairs/va-platform-cop-frontend

File: src/applications/pensions/components/NoFormPage.jsx [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+            <va-link
+              download
+              filetype="PDF"
+              href="https://www.vba.va.gov/pubs/forms/VBA-21P-527EZ-ARE.pdf"
+              pages={8}
+              text="Download VA form 21P-527EZ"
+            />
+            <p>
+              We’ve captured your intent to file date of
+              <strong> XX/XX/XXXX</strong>. You have 12 months from that date to
```
[the remark this answers is not in the document]
Comment (Midge-dev) [reply src/applications/pensions/components/NoFormPage.jsx:83]: Currently it is - we opted to leave it there while we figure out how to capture ITF and then update this place holder accordingly before it goes live. This is just for review purposes and will not stay here.

[the remark this answers is not in the document]
Comment (tblackwe) [reply src/applications/pensions/components/NoFormPage.jsx:83]: ty for the clarification

**Comment by Midge-dev** [comment]
@tblackwe I was having branch issues so I just made a new one and a new PR

https://github.com/department-of-veterans-affairs/vets-website/pull/25717

**Pull request closed without merging**